src/main/java/seedu/address/storage/JsonAdaptedAssignment.java

```diff
@@ -46,15 +45,22 @@ public Assignment toModelType() throws IllegalValueException {
         if (assignmentName == null) {
```

Comment:
maybe it would be better if like isValidName also checks whether its null?

This would shorten the code and make it more readable as it reduces the number of IF statements

Reply:
The two cases are meant to throw different exception messages though, so may not be able to differentiate it that way